### DISCUSSION TITLE
could this work with Elevenlabs ?

### DISCUSSION THREAD

**Discussion by pumayton** [jing332/tts-server-android#112] [Q&A]
Very Useful app, i use it a lot. Many thanks for it! 
Anyone knows would it be possible to configure to use Elevenlabs.io ai voices?  
(using an api key,  its expensive, but very good voices for some cases)
Thanks !

**Comment by jing332** [comment]
Hi @pumayton 
Can you provide an api key for me to testing?

**Reply by pumayton** [reply]
Hello,   I no longer have it, I switched to another service that incorporates natively elevenlabs.   
Thanks for replay .

**Reply by johnnymo87** [reply]
@pumayton I'm curious, which service natively incorporates elevenlabs?

**Reply by pumayton** [reply]
i meant service like moemate : claude, chatgpt, sdxl ,  elevenlabs

**Reply by thorn11166** [reply]
Was going to play with this and see what I could do but I failed 😔. They do offer a free account for testing with API access if you want to give it a go.
